airflow/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -954,3 +954,13 @@ def _list_request(
                 return results
             else:
                 next_token = response["NextToken"]
+
+    def find_processing_job_by_name(self, processing_job_name: str) -> bool:
+        "Query processing job by name"
```

Comment:
```suggestion
        """Query processing job by name"""
```
To fix static checks

Reply:
Fixed.